Movie_MVP/Movie_MVP/DescriptionModule/View/DescriptionViewCell.swift

```diff
@@ -421,11 +421,28 @@ final class DescriptionViewCell: UITableViewCell {
         collectionView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor).isActive = true
     }
 
-    private func loadPhoto(_ movie: Movie, _ photoCacheService: PhotoCacheServiceProtocol?) {
-        let posterPathImage = photoCacheService?.fetchPhoto(byUrl: movie.posterPath)
-        let backdropImage = photoCacheService?.fetchPhoto(byUrl: movie.backdropPath ?? "")
-        backdropImageView.image = backdropImage
-        posterImageView.image = posterPathImage
+    private func loadPhoto(_ movie: Movie, _ imageService: ImageServiceProtocol?) {
+        imageService?.getImage(byUrl: movie.posterPath) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(data):
+                guard let imageData = data else { return }
+                self.backdropImageView.image = UIImage(data: imageData)
+            case let .failure(error):
+                self.alertDelegate?.showAlert(error: error)
+            }
+        }
+
+        imageService?.getImage(byUrl: movie.backdropPath ?? "") { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(data):
+                guard let imageData = data else { return }
+                self.posterImageView.image = UIImage(data: imageData)
+            case let .failure(error):
+                self.alertDelegate?.showAlert(error: error)
+            }
```

Comment:
```suggestion
        imageService?.getImage(byUrl: movie.backdropPath ?? "") { [weak self] result in
            guard let self = self else { return }
            switch result {
            case let .success(data):
                guard let imageData = data else { return }
                self.posterImageView.image = UIImage(data: imageData)
            case let .failure(error):
                self.alertDelegate?.showAlert(error: error)
            }
```

может вынести в другой метод? получается что два запроса в одном не очень